part9/patientor-backend/src/services/patientService.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import patientData from "../../data/patients";
+import toNewPatientEntry from "../utils";
+import { Patient, newPatientEntry, NonSensitivePatientEntry } from "../types";
+
+const patients: Patient[] = patientData.map((obj) => {
+  const object = toNewPatientEntry(obj) as Patient;
+  object.id = obj.id;
+  return object;
+});
```

Comment:
It īpaši patientData ir jābūt JSON, kad redzu, ka tu šeit tāpat ej pāri un mapo ar pārseriem.